src/components/products/product.styled.ts

```diff
@@ -0,0 +1,42 @@
+import styled from 'styled-components';
+
+export const ProductContainer = styled.div`
+    display: flex;
+    flex-direction: row;
+    height: 300px;
+    margin-bottom: 30px;
+`;
+
+export const ProductPhoto = styled.img`
+    width: 300px;
+    height: 300px;
+    border-radius: 30px;
+`;
+
+export const GameInfoContainer = styled.div`
```

Comment:
rename `GameInfoContainer` to be less specific. 
as you are displaying products, let it be `ProductInfoContainer` or something like that.